packages/orm/src/EntityManager.ts

```diff
@@ -1427,7 +1427,6 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW, TX ext
     // Make sure two ReactiveQueryFields don't ping-pong each other forever
     let hookLoops = 0;
     let now = getNow();
-    this.#rm.clearSuppressedTypeErrors();
```

Comment:
note: since errors in reactive fields are immediately populated rather than deferring to `recalcRelationsPendingAssignedIds`, this was clearing out the errors prematurely.  So now we just clear at the end of flush.